### ISSUE TITLE
Use external SRAM if on board present

### ISSUE THREAD

**Issue by cyberman54** [cyberman54/ESP32-Paxcounter#80]
Some ESP32 boards have an external SRAM chip.
Make this usable to have more RAM for MAC buffer container.

**Comment by cyberman54** [comment]
implemented in v1.3.83

**Comment by cyberman54** [comment]
Reopened: include PSRAM check in code still to be done

1)  build flag correctly set, corresponding to ESP IDF menu settings?

2) in main.cpp:

```
if(psramFound()){
ESP.LOGI(TAG, "PSRAM was found and louded");
}
```
